src/ontology/metadata/mondo.sssom.config.yml

```diff
@@ -5,27 +5,27 @@ curie_map:
   DOID: http://purl.obolibrary.org/obo/DOID_
   EFO: http://www.ebi.ac.uk/efo/EFO_
   HGNC: "http://identifiers.org/hgnc/"
-  HGNC__2: "https://identifiers.org/hgnc:"
-  HGNC__3: "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:"
-  HGNC_symbol: https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/
+  # HGNC__2: "https://identifiers.org/hgnc:"
+  # HGNC__3: "https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/HGNC:"
+  # HGNC_symbol: https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/
   HP: http://purl.obolibrary.org/obo/HP_
-  SCTID: http://identifiers.org/snomedct/
-  SCTID__2: http://snomed.info/id/
+  # SCTID: http://identifiers.org/snomedct/
+  # SCTID__2: http://snomed.info/id/
   OMIM: https://omim.org/entry/
-  OMIM__2: http://identifiers.org/omim/
-  OMIM__3: http://purl.obolibrary.org/obo/OMIM_
-  OMIM__4: http://omim.org/entry/
-  MESH: http://identifiers.org/mesh/
+  # OMIM__2: http://identifiers.org/omim/
+  # OMIM__3: http://purl.obolibrary.org/obo/OMIM_
+  # OMIM__4: http://omim.org/entry/
+  mesh: http://identifiers.org/mesh/
   Orphanet: http://www.orpha.net/ORDO/Orphanet_
-  Orphanet__2: https://www.orpha.net/ORDO/Orphanet_
-  Orphanet__3: http://purl.obolibrary.org/obo/Orphanet_
+  # Orphanet__2: https://www.orpha.net/ORDO/Orphanet_
+  # Orphanet__3: http://purl.obolibrary.org/obo/Orphanet_
   oboInOwl: http://www.geneontology.org/formats/oboInOwl#
   NCBITaxon: http://purl.obolibrary.org/obo/NCBITaxon_
   skos: http://www.w3.org/2004/02/skos/core#
   ICD10CM: http://purl.bioontology.org/ontology/ICD10CM/
-  ICD10CM__2: https://icd.codes/icd10cm/
-  ICD10WHO: https://icd.who.int/browse10/2019/en#/
```

Comment:
@hrshdhgd @matentzn I'm reviewing this post merge now.

Just wondering why the ICD10WHO prefix maps removed.